src/modules/alwaysontop/AlwaysOnTop/AlwaysOnTop.cpp

```diff
@@ -315,13 +316,21 @@ bool AlwaysOnTop::IsTopmost(HWND window) const noexcept
     return (exStyle & WS_EX_TOPMOST) == WS_EX_TOPMOST;
 }
 
+bool AlwaysOnTop::IsPinned(HWND window) const noexcept
+{
+    auto handle = GetProp(window, NonLocalizable::WINDOW_PROP);
+    return (handle != NULL);
+}
+
 bool AlwaysOnTop::PinTopmostWindow(HWND window) const noexcept
 {
+    SetProp(window, NonLocalizable::WINDOW_PROP, (HANDLE)1);
```

Comment:
This can fail due to Priveleges, right?
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setpropa#return-value
Let's add some logging for this set.
I don't think the Get needs it since I assume that'll be called too many tiles, but I think it's worth it here.